src/boot/boot.c

```diff
@@ -1894,11 +1894,13 @@ static void config_add_entry_osx(Config *config) {
 }
 
 #if defined(__i386__) || defined(__x86_64__) || defined(__arm__) || defined(__aarch64__)
-static EFI_STATUS boot_windows_bitlocker(void) {
+static EFI_STATUS call_boot_windows_bitlocker(const BootEntry *entry, EFI_FILE *root, EFI_HANDLE parent_image) {
         _cleanup_free_ EFI_HANDLE *handles = NULL;
         size_t n_handles;
         EFI_STATUS err;
 
+        assert(entry);
```

Comment:
Not bad, but this assertion is not necessary in the commit `boot: make .call() method of BootEntry more complete`.